Modules/Loadable/Markups/VTKWidgets/vtkSlicerAngleRepresentation2D.cxx

```diff
@@ -148,7 +148,6 @@ void vtkSlicerAngleRepresentation2D::BuildArc()
   double vector2[3] = { p2[0] - c[0], p2[1] - c[1], p2[2] - c[2] };
   double l1 = vtkMath::Normalize(vector1);
   double l2 = vtkMath::Normalize(vector2);
-  double angle = acos(vtkMath::Dot(vector1, vector2));
```

Comment:
@lassoan  This is the only one worth checking, did you intended to use the angle variable ?

Reply:
Angle measurement was moved to the MRML node, so it is not needed here anymore.